soc/xtensa/intel_adsp/common/include/intel_adsp_ipc.h

```diff
@@ -54,8 +54,9 @@ typedef bool (*intel_adsp_ipc_handler_t)(const struct device *dev, void *arg,
  *
  * @param dev IPC device.
  * @param arg Registered argument from intel_adsp_ipc_set_done_handler().
+ * @return True if external IPC completion will be done, otherwise false.
```

Comment:
pls be more verbose, that true/false means? What need to be done in case of true/false?

Reply:
Yeah I will describe it better. What I meant is:
when function (handler) returns "true", Zephyr IPC API will skip the step of writing to TDR and IDA registers and will not signal IPC message completion in the interrupt context. If returns false, Zephyr IPC will proceed normally.

Reply:
Is this understandable now?